setup.py

```diff
@@ -32,6 +35,15 @@
 tf_cpu = 'tensorflow'
 tf_gpu = 'tensorflow-gpu'
 
+
+def latest_version(package_name):
+    url = "https://pypi.python.org/pypi/%s/json" % (package_name,)
+    data = json.load(request.urlopen(url))
+    versions = [v for v in data["releases"].keys() if not re.search('[a-z]',v)]
```

Comment:
what does the filter do?  is it meant to filter out nightly, release candidates, etc? could you add a comment?
Would it be more robust to search on `if re.search("^[0-9.]+$")` instead?

Reply:
Done